src/plugins/home/server/services/sample_data/data_sets/logs/saved_objects.ts

```diff
@@ -1435,7 +1435,7 @@ export const getSavedObjects = (): SavedObject[] => [
   },
   {
     attributes: {
-      description: '',
+      description: 'View and compare different chart and visualization types',
```

Comment:
should we support i18n for this description?

Reply:
None of the other objects in here have one. This is a copy update, dont want to club a refactor in the PR too since this is for 2.16